### DISCUSSION TITLE
Integration tests

### DISCUSSION THREAD

**Discussion by melezhik** [deepnoodle-ai/risor#4] [Show and tell]
Hi! I am building a free CI service. It could be used for integration tests, for example - https://ci.sparrowhub.io/report/1785 , what do you guys think?

**Comment by melezhik** [comment]
btw test fails right now, not sure why - "cannot provide both a script file and stdin input" in attempt to run some hello world scenario

**Reply by myzie** [reply]
Thanks @melezhik - I'll take a look at both sparrowhub and that test issue. The sparrowhub colorized output is handy.

Regarding the failing test, it could be these lines not working right in the CI environment:
https://github.com/cloudcmds/tamarin/blob/main/tamarin.go#L45-L50

**Reply by melezhik** [reply]
Sure. Please keep in loop. I’ll be happy to help with sparrow if you decide to try it out …
